### ISSUE TITLE
UI refresh to fit the style of ChromeOS

### ISSUE THREAD

**Issue by BugsNash** [GoogleChromeLabs/text-app#345]
Since this is a default app shipped with ChromeOS it should adhere to the ChromeOS visual style.

**Comment by BugsNash** [comment]
Blocked on #369.

**Comment by thibaultmol** [comment]
yeah, this is really needed. Also because you can't use the method of holding down with your mouse on the 'maximize' button in the top right to drag left or right and put it on that side of the screen

**Comment by BugsNash** [comment]
thibaultmol fixing that requires that we use the ChromeOS standard menu bar. If we do that however when the user clicks close on the menu bar we can't listen to this and block closing, so we can't warn the user of unsaved progress (i.e. we would regress on #273). Therefore this issue is blocked on #20.

**Comment by BugsNash** [comment]
Blocked on #472
